packages/app-admin-ui/client/pages/Item/ItemTitle.js

```diff
@@ -29,17 +29,21 @@ export const ItemTitle = memo(function ItemTitle({ titleText }) {
       <PageTitle>{titleText}</PageTitle>
       <ListDescription text={list.adminDoc} />
       <FlexGroup align="center" justify="space-between" css={{ marginBottom: '0.9rem' }}>
-        <div>
-          <IconButton
-            variant="subtle"
-            icon={ChevronLeftIcon}
-            to={list.getFullPersistentPath()}
-            css={{ marginLeft: -12 }}
-          >
-            {list.label}
-          </IconButton>
-          <Search list={list} />
-        </div>
+        {!list.adminConfig.singleton ? (
```

Comment:
I am sure we can handle this differently as this will make plugin dependent on UI and not generic feature of UI being changed by plugin.

I need some suggestion on what that alternate way would be for 
* hiding search field and add new button